SchoolBase.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace School
+{
+    public abstract class SchoolBase : Sorting
+    {
+        public List<Human> _humans = new List<Human>();
+
+        protected string _name;
+
+        public void AddMember(Human human)
+        {
+            _humans.Add(human);
+        }
+
+        public void ShowMembers()
+        {
+            var i = 1;
```

Comment:
If you need an index then I'd recommend using a regular for loop.

Reply:
ok